src/main/java/net/neoforged/neoforge/client/event/ClientPauseUpdatedEvent.java

```diff
@@ -18,7 +18,11 @@
  *
  * <p>This event is fired on the {@linkplain NeoForge#EVENT_BUS main Forge event bus},
  * only on the {@linkplain LogicalSide#CLIENT logical client}.</p>
+ *
+ * @deprecated use {@linkplain ClientPauseChangeEvent.Post} instead
+ * @see ClientPauseChangeEvent
  */
+@Deprecated(forRemoval = true)
```

Comment:
You can remove this class entirely since this will be merged as a breaking change in 1.20.5.